quic-api.md

```diff
@@ -0,0 +1,179 @@
+# QUIC
+
+QUIC is a new, transport layer protocol. It has been recently standardized in [RFC 9000](https://www.rfc-editor.org/rfc/rfc9000.html). It uses UDP as an underlying protocol and it's inherently secure as it mandates TLS 1.3 usage, see [RFC 9001](https://www.rfc-editor.org/rfc/rfc9001.html). Another interesting difference from well known transport protocols as TCP and UDP is that it offers stream multiplexing over a shared connection. The connection between two endpoints negotiates secrets and settings and then the individual streams exchange data between the peers. This allows to have multiple, concurrent, detached data streams that do not affect each other.
```

Comment:
It might be only me, but "detached" sounds more like a stream is detached from a connection... Maybe "independent" will sound better

Reply:
Done